src/python/tests/storage/foglamp_test_storage_init.sql

```diff
@@ -0,0 +1,4 @@
+DELETE FROM foglamp.statistics;
+INSERT INTO foglamp.statistics ( key, description, value, previous_value )
+    VALUES ( 'TEST_1',   'Testing the storage service data 1', 10, 2 ),
+           ( 'TEST_2',   'Testing the storage service data 2', 15, 2 );
```

Comment:
Add comments here as well - "once FOGL-510 is done", we should delete this file

Reply:
No, We wont delete this. We cant insert test data to test storage layer via storage layer. FOGL-510 will migrate fixtures at a common place only.

Reply:
should not we use configuration table or some other which shall allow to have JSON col too, To keep current stuff as is; let's keep statistics too;

Reply:
Done, added statistics data. This will help to write tests for JSON column data too for which stubs are added

Reply:
We have an issue tat we can not currently create test table structures and data inside the storage system using the storage layer and thus we have to bypass the storage layer and go direct. This means we are tied to testing a particular plugin at the moment.

Ultimately we should remove this and have the facilities we need via the storage layer and thus allow the tests to be agnostic of the plugin that is used.